test/views/templates/TemplateBaseSpec.scala

```diff
@@ -16,20 +16,16 @@
 
 package views.templates
 
+import controllers.ControllerBaseSpec
 import mocks.MockAppConfig
 import org.jsoup.Jsoup
-import org.scalamock.scalatest.MockFactory
-import org.scalatestplus.play.guice.GuiceOneAppPerSuite
-import play.api.i18n.{Lang, Messages, MessagesApi, MessagesImpl}
-import play.api.test.Injecting
+import play.api.i18n.MessagesApi
 import play.twirl.api.Html
-import uk.gov.hmrc.play.test.UnitSpec
 
-class TemplateBaseSpec extends UnitSpec with MockFactory with GuiceOneAppPerSuite with Injecting {
+class TemplateBaseSpec extends ControllerBaseSpec {
```

Comment:
Similar comment here as above for the models - I don't think `ControllerBaseSpec` and all of its components are relevant here.